source/documentation/administration_guide/chap-Renewing_certificates.adoc

```diff
@@ -7,28 +7,38 @@
 
 [WARNING]
 ====
-Do _NOT_ let certificates expire. If they expire, the environment becomes non-responsive and recovery is an error prone and time consuming process. If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates.
+Do not let certificates expire. If they expire, the host and {hypervisor-shortname} stop responding, and recovery is an error-prone and time-consuming process. If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates.
 ====
 
 .Procedure
 
-. In the Administration Portal, click menu:Compute[Hosts].
-. Migrate the virtual machines away from the host.
-. Click menu:Management[Maintenance] and btn:[OK].
-. Once the host is in maintenance mode, click menu:Installation[Enroll Certificate].
-. Once enrollment is complete, click menu:Management[Activate].
+. Renew the host certificates:
+.. In the Administration Portal, click menu:Compute[Hosts].
+.. Migrate the virtual machines away from the host.
+.. Click menu:Management[Maintenance] and btn:[OK].
+.. Once the host is in maintenance mode, click menu:Installation[Enroll Certificate].
```

Comment:
Remove "Once the host is in maintenance mode, ". You can't enroll a certificate if the host isn't in maintenance mode, so we do not need the qualifier.

Reply:
it's better this way since it can potentially be a long time before it completes move to maintenance. sometimes requiring user's action to resolve remaining VMs.  Perhaps "Once the host in in maitnenance mode when there are no VMs left on this host"?

Reply:
@michalskrivanek 
In that case, I think we should rephrase this because "Once the host is in maintenance mode" does not imply a long wait or additional actions. 

Suggestion: "When no VMs are left on this host, click menu:Installation[Enroll Certificate]."

That would cover manual intervention as well as automatic live migration.